apps/01PigLatin.js

```diff
@@ -9,15 +9,50 @@ var rl = readline.createInterface({
 
 
 function pigLatin(word) {
+  word = word.toLowerCase();
+  var firstLetter = word[0];
+  console.log(firstLetter);
```

Comment:
It's common to remove `console.log` statements when you're finished debugging and ready to commit code.